src/__tests__/get-state-test.js

```diff
@@ -1,7 +1,7 @@
 import getState from 'get-state.js';
```

Comment:
GLOBAL NIT: Can you clean up the existing ones with `.js` in imports too? Thanks!